app/src/main/kotlin/com/wire/android/ui/calling/controlbuttons/SpeakerButton.kt

```diff
@@ -1,30 +1,41 @@
 package com.wire.android.ui.calling.controlbuttons
 
-import androidx.compose.foundation.Image
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.material3.Icon
 import androidx.compose.material3.IconButton
-import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
 import androidx.compose.ui.tooling.preview.Preview
 import com.wire.android.R
-import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.common.dimensions
 
 @Composable
 fun SpeakerButton(
     isSpeakerOn: Boolean,
     onSpeakerButtonClicked: () -> Unit
 ) {
     IconButton(
-        modifier = Modifier
-            .width(MaterialTheme.wireDimensions.defaultCallingControlsSize)
-            .height(MaterialTheme.wireDimensions.defaultCallingControlsSize),
-        onClick = onSpeakerButtonClicked
+        modifier = Modifier.size(dimensions().defaultCallingControlsSize),
+        onClick = {}
     ) {
-        Image(
+        Icon(
+            modifier = Modifier
+                .wrapContentSize()
+                .clickable(
+                    interactionSource = remember { MutableInteractionSource() },
+                    indication = rememberRipple(bounded = false, radius = dimensions().defaultCallingControlsSize / 2),
```

Comment:
why not define the size / 2 inside the dimensions ?

Reply:
I don't want to be fixed because if we change `defaultCallingControlsSize` later we may forgot to change this one